Cannot find module package.json

I'm seeing this weird issue on v1.0.3. It's not reproducible on v0.3.1

```
Error: The Broccoli Plugin: [BroccoliMergeTrees: Addon#compileAddon(ember-attacher) ] failed with:
Error: Cannot find module '/project/node_modules/ember-attacher/node_modules/babel-plugin-filter-imports/lib//package.json'
```

The integration code is here, which looks perfectly normal to me:

https://github.com/kybishop/ember-attacher/blob/a2f1cd0850c4afc6b371cf121ff35138e4c0eca3/index.js#L35-L43

Any help would be appreciated. Thanks

--- a/src/index.js
+++ b/src/index.js
@@ -1,9 +1,10 @@
 import _ from 'lodash'
+import path from 'path'
 
 import getSpecifiersForRemoval from './getSpecifierNames'
 import removeReferences from './removeReferences'
 
-export default () => ({
+module.exports = () => ({
   visitor: {
     ImportDeclaration: (path, { opts }) => {
       const { imports, keepImports = false } = opts
@@ -31,4 +32,6 @@ export default () => ({
   },
 })
 
-export const baseDir = () => __dirname
+// Provide the path to the package's base directory for caching with broccoli
+// Ref: https://github.com/babel/broccoli-babel-transpiler#caching
+module.exports.baseDir = () => path.resolve(__dirname, '..')

